### PR TITLE
Jonmv/basic http tests

### PR DESCRIPTION
**Pull request by jonmv**
@kkraune or @hakonhall please review and merge.

### CODE REVIEW

File: vespa-cloud/album-recommendation/src/test/application/tests/hit.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "root": {
+        "children": [
+            {
+                "fields": {
+                    "artist": "Foo Fighters",
+                    "album": "Foo Fighters"
+                }
+
```
Comment (hakonhall) [inline vespa-cloud/album-recommendation/src/test/application/tests/hit.json:9]: Remove stray empty line

File: vespa-cloud/album-recommendation/src/test/application/tests/staging-setup/feed-without-stemming.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "name": "Feed without stemming, prior to upgrade",
+    "comment": "Assume current deployment has no stemming on the album field; run this setup",
+    "steps": [
+        {
+            "name": "feed foo",
+            "request": {
+                "cluster": "default",
+                "method": "POST",
+                "uri": "/document/v1/test/music/docid/foo",
+                "body": "../foo.json"
```
Comment (hakonhall) [inline vespa-cloud/album-recommendation/src/test/application/tests/staging-setup/feed-without-stemming.json:11]: Should we avoid dotting up the directory tree in the official examples, as it requires understanding which forefather directory will not be available?  Best practice is to make subdirectories right?  Verify the files doesn't refer to files outside the packaged tree in vespa cli?

File: vespa-cloud/album-recommendation/src/test/application/tests/staging-setup/feed-without-stemming.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Feed without stemming, prior to upgrade",
+    "comment": "Assume current deployment has no stemming on the album field; run this setup",
```
Comment (hakonhall) [inline vespa-cloud/album-recommendation/src/test/application/tests/staging-setup/feed-without-stemming.json:3]: nice

**Review by hakonhall — APPROVED** [review]

File: vespa-cloud/album-recommendation/src/test/application/tests/staging-setup/feed-without-stemming.json (hunk 2 above)
Comment (jonmv) [reply vespa-cloud/album-recommendation/src/test/application/tests/staging-setup/feed-without-stemming.json:11]: The point was to show how staging-setup and staging-test could reference shared resources. Hmm ...

File: vespa-cloud/album-recommendation/src/test/application/tests/hit.json (hunk 1 above)
Comment (jonmv) [reply vespa-cloud/album-recommendation/src/test/application/tests/hit.json:9]:
```suggestion
```

File: vespa-cloud/album-recommendation/src/test/application/tests/staging-setup/feed-without-stemming.json (hunk 2 above)
Comment (jonmv) [reply vespa-cloud/album-recommendation/src/test/application/tests/staging-setup/feed-without-stemming.json:11]: We could add validation. We could also reconsider. Merging now to follow other PRs.

Comment (hakonhall) [reply vespa-cloud/album-recommendation/src/test/application/tests/staging-setup/feed-without-stemming.json:11]: OK, that's a good use-case too, never mind then.

**Pull request merged**